packages/wrangler/src/config/validation.ts

```diff
@@ -3200,6 +3200,47 @@ const validateConsumer: ValidatorFn = (diagnostics, field, value, _config) => {
 	return isValid;
 };
 
+const isValidDate = (value: string): boolean => {
```

Comment:
We have the `validation-helpers` file - could this be moved into there? I know it's not an ideal spot, as this is a really very generic helper and we don't have a generalised "utils" folder, but maybe it will help for discoverability if it's in a dedicated helper file.

Reply:
✅